kad-dht/README.md

```diff
@@ -268,23 +268,55 @@ heuristic of the value to make the decision.
 
 ### Content provider advertisement and discovery
 
-Nodes must keep track of which nodes advertise that they provide a given key
-(CID). These provider advertisements should expire, by default, after 24 hours.
-These records are managed through the `ADD_PROVIDER` and `GET_PROVIDERS`
+There are two things at play with regard to provider record (and therefore content)
+liveness and reachability:
+
+Content providers need to make sure that their content is reachable, despite peer churn
+and nodes that store and serve provider records need to make sure that the Multihashes whose 
```

Comment:
```suggestion
and nodes that store and serve provider records need to make sure that the CID whose 
```

Isn't this a CID?

Reply:
the keys for provider records are multihashes (e.g. https://github.com/libp2p/go-libp2p-kad-dht/blob/dae5a9a5bd9c7cc8cfb5073c711bc308efad0ada/routing.go#L431)

TLDR on why:
- You are using provider records as a rendezvous point for all the parties who have advertised they have some data
- The same multihash can be in different CIDs (e.g. CIDv0 vs CIDv1 of a SHA-256 dag-pb object, or the same multihash but with different codecs such as dag-pb vs raw)
- This means you want to rendezvous on the minimal thing everyone agrees on, which is the multihash